api/common/v1alpha1/ref.go

```diff
@@ -71,6 +71,17 @@ type TargetRef struct {
 	// all data plane types are targeted by the policy.
 	// +kubebuilder:validation:MinItems=1
 	ProxyTypes []TargetRefProxyType `json:"proxyTypes,omitempty"`
+	// Namespace specifies the namespace of target resource. If empty only resources in policy namespace
+	// will be targeted.
+	// Namespace can be used only with MeshService kind, and when targeting resources in Kubernetes zone.
+	Namespace string `json:"namespace,omitempty"`
+	// Labels are used to select group of MeshServices that match labels. Either Labels or
+	// Name and Namespace can be used.
+	// Labels can be used only with MeshService kind.
+	Labels map[string]string `json:"labels,omitempty"`
+	// SectionName is used to target specific port from MeshService.ports[]. Only traffic to this port will be affected.
```

Comment:
In GAPI docs [SectionName](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.SectionName) is not limited to port, it can select section of various resources. I'd give it a generic comment and added MeshService's port as an example. 

Also, we probably should mention it's not a port value but actually a port name

Reply:
changed